aws/rust-runtime/aws-credential-types/README.md

```diff
@@ -0,0 +1,10 @@
+# aws-credential-types
+
+This crate provides the items concerned with AWS SDK credentials including:
+* A trait for credential providers
```

Comment:
In this context, we've been using the plural for "credentials" since the provider provides multiple credentials (access key, secret key, and sometimes a token):
```suggestion
* A trait for credentials providers
```